.jsbeautifyrc

```diff
@@ -6,5 +6,8 @@
     },
     "scss": {
         "indent_size": 2
+    },
```

Comment:
cho file này vào ignore đi nhé, vì cái này là liên quan đến editor cá nhân không phải của toàn project